src/views/users/Users.vue

```diff
@@ -88,11 +136,36 @@ export default {
       retreiveProfiles('Newest');
     });
 
+    const paginationCallback = (page) => {
+      const newPagination = {
+        ...pagination.value,
+        currentPage: page - 1,

```

Comment:
Bai, we'll start practicing good naming conventions. So as an example this is a users listing page, so we will replace
"transDataList" with "usersDataList"